pom.xml

```diff
@@ -333,6 +340,13 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-inline</artifactId>
+                <version>3.12.4</version>
```

Comment:
Given that this version will always be the same as the one in `mockito-core`, you should extract them both to a `mockito.version` property.